packages/patterns/package.json

```
@@ -0,0 +1,53 @@
+{
+	"name": "@wordpress/patterns",
+	"version": "1.0.0",
+	"description": "Management of user pattern editing.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"gutenberg",
+		"patterns"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/HEAD/packages/patterns/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/editor"
```

Should this be `patterns`?

```suggestion
		"directory": "packages/patterns"
```

---

ah, well spotted - thanks